src/util/schema.py

```diff
@@ -345,6 +345,9 @@ class ExecPlanCoordsModel(BaseModel):
 
     sequence: int = Field(..., description="Sequence index.")
     position: int = Field(..., description="Position within the sequence.")
+    internal: Optional[bool] = Field(
+        False, description="Using internal liquidity"
+    )
```

Comment:
So it's optional but defaults to false? Is there a scenario where it's actually None? Or is this just for backwards compatibility?